docs/setting-up/installing.any

```diff
@@ -295,8 +295,7 @@ a tool that meshes well with Concourse's ideals, buckle up and head over to
       CONCOURSE_BASIC_AUTH_USERNAME: concourse
       CONCOURSE_BASIC_AUTH_PASSWORD: changeme
       CONCOURSE_EXTERNAL_URL: "$\{CONCOURSE_EXTERNAL_URL\}"
-      CONCOURSE_POSTGRES_DATA_SOURCE: |
```

Comment:
How about adding a `-` to the end of this instead? The value's a bit long otherwise; it'd run off the side of this page:

https://concourse.ci/docker-repository.html